elasticdl/python/worker/worker.py

```diff
@@ -126,10 +124,7 @@ def _init_embedding_layer(self):
         """
         self._embedding_layers = find_layer(self._model, Embedding)
         for layer in self._embedding_layers:
-            layer.set_lookup_func(self.lookup_embedding)
-        if self._embedding_layers:
-            # TODO check that Redis IP/PORT is set
```

Comment:
Perhaps do not remove this TODO?

Reply:
The new logic is that

```python
for layer in self._embedding_layers:
    layer.set_endpoint(self._embedding_service_endpoint)
```
It set redis IP/Port to embedding layer directly.